<launch>

    <node pkg="screwed_planning"
          type="simple_planner_node.py"
          name="simple_planner"
          output="screen">

    </node>

</launch>